components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -3268,6 +3268,186 @@ paths:
           source: 'curl -k -X PUT -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
           -F file=@theme.zip "https://127.0.0.1:9443/api/am/admin/v4/tenant-theme"'
 
+  ######################################################
+  # Organization theme of Devportal related APIs
+  ######################################################
+  /org-themes:
+    get:
+      operationId: getOrgThemes
+      summary: Retrieve UUIDs of org-theme contents
+      description: Returns the UUIDs of org-theme contents and their publish status.
+      parameters:
+        - name: publish
+          in: query
+          description: Filter themes based on published status
+          required: false
+          schema:
+            type: boolean
+      responses:
+        200:
+          description: List of org themes
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ContentPublishStatusResponse'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+      security:
+        - OAuth2Security:
+            - apim:admin
+            - apim:tenant_theme_manage
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -X GET -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+                "https://127.0.0.1:9443/api/am/admin/v4/org-themes?publish=true"'
+
+    post:
+      operationId: importOrgTheme
+      summary: Import org theme
+      description: Imports a drafted zip of an org theme to APIM.
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema:
+              required:
+                - file
+              properties:
+                file:
+                  type: string
+                  description: |
+                    The org-content.zip file
+                  format: binary
+      responses:
+        200:
+          description: Successfully imported
+        400:
+          $ref: '#/components/responses/BadRequest'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        413:
+          $ref: '#/components/responses/PayloadTooLarge'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+      security:
+        - OAuth2Security:
+            - apim:admin
+            - apim:tenant_theme_manage
```

Comment:
with this, we have to update the tenant-conf.json to attach the scope and add new scopes at the securitySchemes section of this file

Reply:
@AnuGayan This apim:tenant_theme_manage scope is not a new scope. It's already there in APIM.

Reply:
ack